src/main/java/com/bookstore/controller/CategoryController.java

```diff
@@ -85,7 +86,7 @@ public void deleteCategory(@PathVariable Long id) {
     })
     @PreAuthorize("hasRole('USER')")
     @GetMapping("/{id}/books")
-    public List<BookDto> getBooksByCategoryIds(@PathVariable Long id, Pageable pageable) {
-        return bookService.getBooksByCategoryIds(id, pageable);
+    public List<BookDto> getBooksByCategoryId(@PathVariable Long id, Pageable pageable) {
```

Comment:
Please, add test for this
```suggestion
    public List<BookDto> getBooksByCategoryId(@Positive @PathVariable Long id, Pageable pageable) {
```